docs/onboard-to-seed.md

```diff
@@ -0,0 +1,54 @@
+# Request for SEED and submit Intune device ID for SEED onboarding
+
+This article is applicable only for public officers who have an existing active TechPass ID and need to onboard their device to SEED. Public officers can request for SEED from the [TechPass portal](https://portal.techpass.gov.sg/public/home) using their non-SE GSIB machine.
+
+>**Notes**:
+>- You can't onboard a GSIB or DWP device to SEED.
+>- If you are a vendor or contractor, contact your project manager or reporting officer to request for SEED provisioning before proceeding with your SEED onboarding.
+
+
+_To onboard your device to SEED:_
+
+1. Go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
+
+<kbd>![log in](assets/images/onboarding/po-non-se/log-in-with-techpass.png ':size=90%')</kbd>
+
+2. Choose your TechPass account.
+
```

Comment:
If you are not logged into your WOG account, you will be prompted to log in, otherwise select your existing session to login with.

Reply:
Added the following before the screenshot
?> Note If you are not logged into your WOG account, you will be prompted to log in to it before signing in to your TechPass account.